src/reporting/reportingToken.se

```diff
@@ -77,6 +77,19 @@ def migrateLosingTokens():
     require(self.market.getFinalWinningReportingToken() != self)
     self.supply = 0
     reputationSupply = self.getReputationToken().balanceOf(self)
+    reputationToken = self.market.getReputationToken()
+    disputeBondToken = self.market.getAutomatedReporterDisputeBondToken()
+    amount = min(disputeBondToken.getBondRemainingToBePaidOut(), reputationToken.balanceOf(self))
```

Comment:
This needs to include REP held by the disputeBondToken but not yet paid out.
```
amountNeeded = disputeBondToken.getBondRemainingToBePaidOut() - reputationToken.balanceOf(disputeBondToken)
amount = min(amountNeeded, reputationToken.balanceOf(self))
```

Same for following few blocks.